### PR TITLE
Component reference for v1.25

### PR DESCRIPTION
**Pull request by tengqm**
This PR batch updates the references for Kubernetes components command line tools, generated using the [gen-compdocs tool](kubernetes-sigs/reference-docs), based on the new release tags for kubernetes v1.25.

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ each Pod in the scheduling queue according to constraints and available
 resources. The scheduler then ranks each valid Node and binds the Pod to a
 suitable Node. Multiple different schedulers may be used within a cluster;
 kube-scheduler is the reference implementation.
-See [scheduling](/docs/concepts/scheduling-eviction/)
+See [scheduling](https://kubernetes.io/docs/concepts/scheduling-eviction/)
```
Comment (jihoon-seo) [inline content/en/docs/reference/command-line-tools-reference/kube-scheduler.md:30]: Would it be possible for us to remove this hardcoded site domain name?

```suggestion
See [scheduling](/docs/concepts/scheduling-eviction/)
```

**Review by jihoon-seo — COMMENTED** [review]
@tengqm PTAL!

File: content/en/docs/reference/command-line-tools-reference/kube-scheduler.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/kube-scheduler.md:30]: Won't fix. This is generated out of upstream source code.

**Comment by sftim** [comment]
@tengqm please set a PR description - it might not be obvious to everyone what this is about.

File: content/en/docs/reference/command-line-tools-reference/kube-scheduler.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/kube-scheduler.md:30]: We _could_ fix the generator code, but it'd be a very low priority thing.

**Comment by tengqm** [comment]
> @tengqm please set a PR description - it might not be obvious to everyone what this is about.

Done.

File: content/en/docs/reference/command-line-tools-reference/kube-scheduler.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/kube-scheduler.md:30]: I don't think make the tool looking smart is a smart thing to do.

**Comment by reylejano** [comment]
Preview: https://deploy-preview-36205--kubernetes-io-main-staging.netlify.app/docs/reference/command-line-tools-reference/kube-apiserver/

**Pull request merged**